client/devdocs/design/index.jsx

```diff
@@ -116,60 +116,60 @@ let DesignAssets = React.createClass( {
 						/> }
 
 				<Collection component={ component } filter={ filter }>
-					<Accordions componentUsageStats={ componentsUsageStats.accordion } />
-					<Banner />
-					<BulkSelect />
-					<ButtonGroups />
-					<Buttons componentUsageStats={ componentsUsageStats.button } />
-					<Cards />
-					<ClipboardButtonInput />
-					<ClipboardButtons />
-					<Count />
-					<CountedTextareas />
-					<DatePicker />
-					<DropZones searchKeywords="drag" />
-					<EllipsisMenu />
-					<EmojifyExample />
-					<EmptyContent />
-					<ExternalLink />
-					<FAQ />
-					<FeatureGate />
-					<FilePickers />
-					<FoldableCard />
-					<FormattedHeader />
-					<FormFields searchKeywords="input textbox textarea radio" />
-					<Gauge />
+					<Accordions componentUsageStats={ componentsUsageStats.accordion } readmeFilePath="accordion" />
```

Comment:
I originally had it just automatically grab the README based off of the name of the component, but realized that the README location isn't consistent for every component.
Some are buried in subfolders, so it requires a little bit more sophistication and I'm not quite sure how that would work.